light-zk.js/tests/utxo.test.ts

```diff
@@ -352,10 +384,49 @@ describe("Utxo Functional", () => {
       verifierProgramLookupTable:
         lightProvider.lookUpTables.verifierProgramLookupTable,
     });
-    if (receivingUtxo1) {
-      Utxo.equal(poseidon, receivingUtxo, receivingUtxo1, true);
+    if (receivingUtxo1Unchecked) {
+      Utxo.equal(poseidon, receivingUtxo, receivingUtxo1Unchecked, true);
     } else {
-      throw new Error("decrypt failed");
+      throw new Error("decrypt unchecked failed");
+    }
+
+    const receivingUtxoNoAes = new Utxo({
+      poseidon,
+      assets: inputs.assets,
+      amounts: inputs.amounts,
+      account: new Account({ poseidon }),
```

Comment:
This is a random account with an aes secret key, thus in line 406 you are encrypting the utxo with aes.

Reply:
so do you suggest to use this instead -> 

```
account: Account.fromPubkey(pubKey, poseidon),
```

?

Reply:
@sergeytimoshin this is still not solved, like matteo wrote this should be Account.fromPubkey(...)